src/bin/helpers.js

```diff
@@ -1,873 +1,974 @@
+		"",
+	);
+
+export const commas = (number) => {
+	const parts = number.toString().split(".");
+	const result =
+		parts.length <= 1
+			? `${parts[0].replace(/\B(?=(\d{3})+(?!\d))/g, ",")}`
+			: `${parts[0].replace(/\B(?=(\d{3})+(?!\d))/g, ",")}.${parts[1]}`;
+	return result;
 };
 
-const whole_wits = (number, digits) => {
-  const lookup = [
-    { value: 1n, symbol: "pedros" },
-    { value: 10n ** 6n, symbol: "mWits" },
-    { value: 10n ** 9n, symbol: " Wits" },
-    { value: 10n ** 12n, symbol: "KWits" },
-    { value: 10n ** 15n, symbol: "MWits" },
-  ]
-  // const regexp = /\.0+$|(?<=\.[0-9])0+$/
-  const item = lookup.findLast(item => number >= item.value)
-  const quotient = item ? Number(BigInt(number) / item.value) : number.toString()
-  const decimals = item ? (item.value + BigInt(number) - BigInt(quotient) * item.value).toString().slice(1) : ""
-  return item ? `${commas(quotient)}${decimals !== "" ? `.${decimals.slice(0, digits)}` : ""} ${item.symbol}` : "(no coins)"
-}
-
-const bblue = (str) => `\x1b[1;98;44m${str}\x1b[0;0;0m`
-const bcyan = (str) => `\x1b[38;46m${str}\x1b[0;0m`
-const bgreen = (str) => `\x1b[30;42m${str}\x1b[0;0m`
-const bred = (str) => `\x1b[30;41m${str}\x1b[0;0m`
-const bviolet = (str) => `\x1b[30;45m${str}\x1b[0;0m`
-
-const lcyan = (str) => `\x1b[1;96m${str}\x1b[0m`
-const lgray = (str) => `\x1b[1;90m${str}\x1b[0m`
-const lgreen = (str) => `\x1b[1;92m${str}\x1b[0m`
-const lmagenta = (str) => `\x1b[1;95m${str}\x1b[0m`
-const lyellow = (str) => `\x1b[1;93m${str}\x1b[0m`
-const mblue = (str) => `\x1b[94m${str}\x1b[0m`
-const mcyan = (str) => `\x1b[96m${str}\x1b[0m`
-const mgreen = (str) => `\x1b[92m${str}\x1b[0m`
-const mmagenta = (str) => `\x1b[0;95m${str}\x1b[0m`
-const mred = (str) => `\x1b[91m${str}\x1b[0m`
-const myellow = (str) => `\x1b[93m${str}\x1b[0m`
-
-const blue = (str) => `\x1b[34m${str}\x1b[0m`
-const cyan = (str) => `\x1b[36m${str}\x1b[0m`
-const gray = (str) => `\x1b[90m${str}\x1b[0m`
-const green = (str) => `\x1b[32m${str}\x1b[0m`
-const magenta = (str) => `\x1b[0;35m${str}\x1b[0m`
-const normal = (str) => `\x1b[98m${str}\x1b[0m`
-const red = (str) => `\x1b[31m${str}\x1b[0m`
-const white = (str) => `\x1b[1;98m${str}\x1b[0m`
-const yellow = (str) => `\x1b[33m${str}\x1b[0m`
-
-function countLeaves (t, obj) {
-  if (!obj || typeof obj === "string") return 0
-  if (obj instanceof t) return 1
-  if (Array.isArray(obj)) return obj.reduce((sum, item) => sum + countLeaves(t, item), 0)
-  else return Object.values(obj).reduce((sum, item) => sum + countLeaves(t, item), 0)
-}
-
-function deleteExtraFlags (args) {
-  return args.filter(arg => !arg.startsWith("--"))
-}
-
-function cmd (...command) {
-  return new Promise((resolve, reject) => {
-    exec(command.join(" "), { maxBuffer: 1024 * 1024 * 10 }, (error, stdout, stderr) => {
-      if (error) {
-        reject(error)
-      }
-      if (stderr) {
-        reject(stderr)
-      }
-      resolve(stdout)
-    })
-  })
+function _toFixedTrunc(x, n) {
```

Comment:
in [`./src/bin/cli/network.js`](https://github.com/witnet/witnet-sdk/pull/68/files#diff-33e2df40c9aea8803525b4192c78e744256ec6a3afaf07407e725df20c055467R530) we are still using `toFixedTrunc`

```suggestion
function toFixedTrunc(x, n) {
```